static/src/javascripts/bootstraps/enhanced/trail.js

```diff
@@ -102,8 +102,13 @@ define([
 
     function repositionComments() {
         if (!identityApi.isUserLoggedIn()) {
+            var $comments = $('.js-comments');
```

Comment:
The definitive references would be [this](https://gist.github.com/paulirish/5d52fb081b3570c81e3a) and [that](https://csstriggers.com/)